src/README.md

```diff
@@ -0,0 +1,136 @@
+# API setup:
+
+## Mongo Database setup:
+
+- Create MongoDB account.
+
+- Create Database cluster.
+
+- Create User collection.
+
+- In the "Database Deployments" overview, press on "Connect".
+
+![alt](https://i.imgur.com/A0Dwk6F.png)
+
+- In the popup press on "Connect your application" and then you will be greeted with the screen which will have connection string defined for you:
+
+![alt](https://i.imgur.com/uUSSwMD.png)
+
+In the above example, "user" is username who has an access to your database cluster, and password is theirs password.
+
+- To register new user who will have an access to the cluster (admin user because), go to the "Database Access" and press on "Add new database user".
+
+![alt](https://i.imgur.com/x6CyUzw.png)
+
+Save password of that user and their username, you will use it in your connection string.
+
+- Finally, in "Network Access" change IP address to 0.0.0.0 to make your Database accessible by any IP (anyone who has connection string with correct user credentials you defined in previous step).
+
+![alt](https://i.imgur.com/A4a8mbQ.png)
+
+- In the project folder, create ".env" file and fill it with your information. Example is here:
+
+![alt](https://i.imgur.com/msHT5eu.png)
+
+environment variable names should be the same, the only parts that change are values.
+
+## Firebase server setup:
+
+- Register account in firebase.
+
+- In your console, "Add new project".
+
+![alt](https://i.imgur.com/WL2jaVO.png)
+
+- When project is created, go to the "Project settings".
+
+![alt](https://i.imgur.com/ZDowUt5.png)
+
+- Go to "Service accounts" and "Generate new private key".
+
+![alt](https://i.imgur.com/PN3Svzt.png)
+
+- Now, save that file on your local server and export its location as the Global variable.
+
+![alt](https://i.imgur.com/kGtzCUI.png)
+
+At this point your server will be able to connect to the Firebase project you created. That being said, there are still some steps to be done to setup authentication part.
+
+- In "Project Overview" find "Authentication" and go to "Sign-in method" tab:
+
+![alt](https://i.imgur.com/FFw2QIH.png)
+
+- On this tab press "Add new provider" and select "Email/Password".
+
+![alt](https://i.imgur.com/bTtm9ub.png)
+
+- Be sure to disable passwordless authentication in the setup settings. You can find it if you press "Edit configuration" of the provider on the "Sign-in method" tab:
+
+![alt](https://i.imgur.com/9bXzMev.png)
+
+Congratulations! Your server side is ready to run.
+
+## Run Server:
+
+To run server install latest npm and run npm install. All required packages will be installed automatically. Now, go to the project directory and run either npm start or npm run start:dev. npm start will run server as usualy, whereas, npm run start:dev will run server with nodemon which allows realtime restart of the server on file changes.
```

Comment:
I would suggest adding backticks `` around code segments to clarify what are executable commands. Example: `npm install`

Reply:
Will be done.